src/main/java/mekanism/common/block/BlockTransmitter.java

```diff
@@ -424,4 +442,24 @@ public TileEntity createNewTileEntity(World worldIn, int meta)
 		
 		return null;
 	}
+	
+	private Optional<TileEntitySidedPipe> getTileEntitySidedPipe(TileEntity entity)
```

Comment:
It's certainly not necessary, I simply used that method because it's what the MCMultipart source does and I found it to feel a bit cleaner as it made null checks a bit more obvious for when they're necessary.

It'd be easy enough for me to refactor it out, if desired